<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RCDMemtransfer</class>
 <widget class="QDialog" name="RCDMemtransfer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Tests</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinRuns">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>310</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelRuns">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Runs:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBenPar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Set benchmarking parameters:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Memory Size (MB):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Lower Index: (Min. 500)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Upper Index: (Max 2047)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinLower">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>500</number>
   </property>
   <property name="maximum">
    <number>2047</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinUpper">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>500</number>
   </property>
   <property name="maximum">
    <number>2047</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chart Type:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinLinear">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelLinear">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Linear Step:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAutofill">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Autofill Sample Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWarning">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>LABEL</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelWarning_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>LABEL</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelWarning_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>180</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>LABEL</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Linear</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
